rest-service/manager_rest/storage/resource_models.py

```diff
@@ -797,6 +797,10 @@ class _Filter(CreatedAtMixin, SQLResourceBase):
 
```

Comment:
Just noticed: the class is called _Filter, rather than BaseFilter. I don't know how standard it is...

Reply:
Yep, it might be worth changing it.